syntax/dirvish.vim

```diff
@@ -3,13 +3,14 @@ if 'dirvish' !=# get(b:, 'current_syntax', 'dirvish')
 endif
 
 let s:sep = exists('+shellslash') && !&shellslash ? '\' : '/'
+let s:sep_escaped = s:sep == '\' ? '\\' : '/'
```

Comment:
nit: let's stick with `_esc` suffix to be consistent with similar variables in the plugin.
```suggestion
let s:sep_esc = s:sep == '\' ? '\\' : '/'
```